src/add-candidate.php

```diff
@@ -42,7 +42,73 @@
                 <link rel="stylesheet" href="../vendor/node_modules/bootstrap/dist/css/bootstrap.min.css" />
                 <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
 
+                <style>
+                    .modal-content {
+                        border-radius: 20px;
+                    }
+
+                    .modal-header {
+                        text-align: center;
+                        /* Center the content */
+                    }
+
+                    .modal-header i {
+                        font-size: 7rem;
+                        /* Make the icon bigger */
+                        display: block;
+                        /* Ensure the icon is on its own line */
+                        margin: auto;
+                    }
+
+                    .modal-body {
+                        text-align: center;
+                    }
+
+                    .modal-body h2 {
+                        color: #333;
+                        font-size: 10px;
+                        margin-bottom: 5px;
+                    }
+
+                    .modal-body p {
+                        font-size: 18px;
+                        color: #555;
+                        margin-bottom: 5px;
+
+                    }
+
+                    .modal-footer {
+                        justify-content: center;
+                        border-top: none;
 
+                    }
+
+                    .cancel {
+                        background-color: lightgray;
+                        color: #B2BEB5;
+                        font-weight: bold;
+                        padding: 5px 18px;
+                    }
+
+                    .discard-button {
+                        background-color: #FFA500;
+                        color: #fff;
+                        padding: 5px 28px;
+                        font-weight: 600;
+                    }
+
+                    /* Button Styles */
+                    .close {
+                        font-size: 1.4rem;
+                        color: #aaa;
+                        opacity: 1;
+                    }
+
+                    .close:hover {
+                        color: #000;
+                        opacity: 1;
+                    }
+                </style>
```

Comment:
Can you have this css in a separate or existing stylesheet file?

Reply:
I tried to separate it and put it in the existing stylesheet. But it wouldn't function if it was separated. Could you suggest any techniques, @Carl-Tabuso ?